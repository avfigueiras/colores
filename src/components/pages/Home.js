import React from 'react';
import ListColors from '../listColors/index';
import Layout from '../Layout/index';

const Home = () => {
    return ( 
        <Layout>
            <ListColors />
        </Layout>
    )
}

export default Home;
