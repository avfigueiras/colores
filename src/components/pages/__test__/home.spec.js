import React from 'react';
import Home from '../Home';
import { render } from '@testing-library/react';

describe("Home test", () =>{
    test("test 1", () =>{
        const homeComponent = render(<Home />);

        expect(homeComponent).toMatchSnapshot();
    })
})