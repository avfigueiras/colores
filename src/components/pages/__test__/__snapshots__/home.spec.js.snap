// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test test 1 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="title"
      >
        <a
          class="link"
          href="/"
        >
          Colores
        </a>
      </div>
      <div
        class="container"
      >
        <div
          class="wrapper"
        >
          <span>
            No existen colores disponibles
          </span>
        </div>
      </div>
      <div
        class="action"
      >
        <div
          class="boton"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root icon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
            />
          </svg>
          <button
            class="btn"
          >
             Anterior
          </button>
        </div>
        <div
          class="boton"
        >
          <button
            class="btn"
          >
             Siguiente
          </button>
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root icon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="title"
    >
      <a
        class="link"
        href="/"
      >
        Colores
      </a>
    </div>
    <div
      class="container"
    >
      <div
        class="wrapper"
      >
        <span>
          No existen colores disponibles
        </span>
      </div>
    </div>
    <div
      class="action"
    >
      <div
        class="boton"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root icon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
          />
        </svg>
        <button
          class="btn"
        >
           Anterior
        </button>
      </div>
      <div
        class="boton"
      >
        <button
          class="btn"
        >
           Siguiente
        </button>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root icon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
          />
        </svg>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
